tsconfig.json

```diff
@@ -2,19 +2,20 @@
   "compilerOptions": {
     "module": "commonjs",
     "lib": [
-      "es6"
+      "es6", "dom"
     ],
     "target": "es6",
     "noImplicitAny": true,
     "noImplicitThis": true,
     "strictFunctionTypes": true,
     "strictNullChecks": true,
+    "skipLibCheck": true,
```

Comment:
It seems that `typescript@4.1.3` released yesterday broke a type def... Still need to check, we have been experiencing a lot of issues with typescript releases changing outputs of ts declaration files generated.